Sources/RInAppMessaging/Extensions/UIApplication+IAM.swift

```diff
@@ -18,7 +18,7 @@ extension UIApplication {
     func getCurrentStatusBarStyle() -> UIStatusBarStyle? {
         if #available(iOS 13.0, *),
            let keyScene = connectedScenes
-                .filter({ $0.activationState == .foregroundActive })
+                .filter({ $0.activationState != .unattached })
```

Comment:
@imaginaris In which case does it occur?
Is it a fix for apps having a SceneDelegate?

Reply:
We shouldn't filter out the background activationState and any other state. I don't fully understand what exactly is a scene with `.unattached` activationState and I don't know how to trigger it. But it seemed like a state that we don't want to support (after looking at its description).